parser-typechecker/src/Unison/Codebase/SqliteCodebase.hs

```diff
@@ -93,28 +91,33 @@ import Unison.Type (Type)
 import qualified Unison.Type as Type
 import qualified Unison.Util.Set as Set
 import qualified Unison.WatchKind as UF
-import UnliftIO (MonadUnliftIO, catchIO, finally, throwIO)
+import UnliftIO (catchIO, finally, MonadUnliftIO, throwIO)
 import qualified UnliftIO
-import UnliftIO.Directory (createDirectoryIfMissing, doesDirectoryExist, doesFileExist)
+import UnliftIO.Directory (createDirectoryIfMissing, doesDirectoryExist, doesFileExist, removePathForcibly)
 import UnliftIO.Exception (bracket, catch)
 import UnliftIO.STM
+import Data.Either.Extra ()
 
 debug, debugProcessBranches, debugCommitFailedTransaction :: Bool
 debug = False
 debugProcessBranches = False
 debugCommitFailedTransaction = False
 
+-- | Prefer makeCodebasePath or makeCodebaseDirPath when possible.
 codebasePath :: FilePath
 codebasePath = ".unison" </> "v2" </> "unison.sqlite3"
 
-v2dir :: FilePath -> FilePath
-v2dir root = root </> ".unison" </> "v2"
+makeCodebasePath :: FilePath -> FilePath
+makeCodebasePath root = makeCodebaseDirPath root </> "unison.sqlite3"
+
+makeCodebaseDirPath :: FilePath -> FilePath
+makeCodebaseDirPath root = root </> ".unison" </> "v2"
```

Comment:
We may need to have a chat about this threshold again at some point, I understand that noise in PRs is annoying, but also, reviewing many trivial PRs becomes noise for both reviewers and the implementer, which results in fewer cleanups happening.

This is a change which is so small that it seems like a waste of time to pull out into a separate PR (all things have a cost), but which is still good to have since it prevents divergence on codebase paths. In this case I figured the relatively small amount of noise was worth it.

Let's chat about this in a tech-chat or one-on-one soon 👍🏼